zebra-consensus/src/block/subsidy/general.rs

```diff
@@ -0,0 +1,173 @@
+//! Block and Miner subsidies, halvings and target space modifiers. - [§7.7][7.7]
+//!
+//! [7.7]: https://zips.z.cash/protocol/protocol.pdf#subsidies
+
+use std::convert::TryFrom;
+
+use zebra_chain::{
+    amount::{Amount, Error, NonNegative},
+    block::Height,
+    parameters::{Network, NetworkUpgrade::*},
+};
+
+use super::founders_reward::founders_reward;
```

Comment:
I'd like to try to remove this dependency.

I'll see what I can come up with today.

Reply:
I made this change in 2aa9a5064dcb9985cdfe15f71d6ed9c66676bbdb in https://github.com/oxarbitrage/zebra/pull/29

Reply:
I made a minor comment about this at https://github.com/oxarbitrage/zebra/pull/29#discussion_r497484941 but i think it will be fine.